Formula/m/mvt.rb

```diff
@@ -43,11 +38,33 @@ class Mvt < Formula
     sha256 "f30c3cb33b24454a82faecaf01b19c18562b1e89558fb6c56de4d9118a032fd5"
   end
 
+  resource "click" do
+    url "https://files.pythonhosted.org/packages/96/d3/f04c7bfcf5c1862a2a5b845c6b2b360488cf47af55dfa79c98f6a6bf98b5/click-8.1.7.tar.gz"
+    sha256 "ca9853ad459e787e2192211578cc907e7594e294c7ccc834310722b41b9ca6de"
+  end
+
   resource "idna" do
     url "https://files.pythonhosted.org/packages/bf/3f/ea4b9117521a1e9c50344b909be7886dd00a519552724809bb1f486986c2/idna-3.6.tar.gz"
     sha256 "9ecdbbd083b06798ae1e86adcbfe8ab1479cf864e4ee30fe4e46a003d12491ca"
   end
 
+  # The `iosbackup` resource requires `nskeyedunarchiver`, so it must be installed first
+  resource "nskeyedunarchiver" do
+    url "https://files.pythonhosted.org/packages/e8/d9/227a00737de97609b0b2d161905f03bb8e246df0498ec5735b83166eef8f/NSKeyedUnArchiver-1.5.tar.gz"
+    sha256 "eeda0480021817336e0ffeaca80377c1a8f08ecc5fc06be483b48c44bad414f4"
+  end
+
+  # The `iosbackup` resource requires `pycryptodome`, so it must be installed first
+  resource "pycryptodome" do
+    url "https://files.pythonhosted.org/packages/b9/ed/19223a0a0186b8a91ebbdd2852865839237a21c74f1fbc4b8d5b62965239/pycryptodome-3.20.0.tar.gz"
+    sha256 "09609209ed7de61c2b560cc5c8c4fbf892f8b15b1faf7e4cbffac97db1fffda7"
+  end
+
+  resource "iosbackup" do
+    url "https://files.pythonhosted.org/packages/db/b8/4cd52322deceb942b9e18b127d45d112c2f7a3ec7821ab528659d4f04275/iOSbackup-0.9.925.tar.gz"
+    sha256 "33545a9249e5b3faaadf1ee782fe6bdfcdb70fae0defba1acee336a65f93d1ca"
+  end
+
```

Comment:
Not sure if there's any better way to handle this constraint.

Reply:
Another option would be to change the install block to install `iosbackup` last:

```ruby
venv = virtualenv_create(libexec, "python3.12")
venv.pip_install resources.reject { |r| r.name == "iosbackup" }
venv.pip_install_and_link resource("iosbackup")
venv.pip_install_and_link buildpath
```

Reply:
That's exactly the sort of thing I was looking for. I will handle that in a follow up PR since the bottles have already been uploaded. Thanks!